course/app/courseapp/model/career.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+from flask import g
+import datetime
+import time
+
+
+class Course:
+    def __init__(self, db_row, subjects):
+        if db_row is not None:
+            self.courseID = db_row[0]
+            self.courseName = db_row[1]
+            self.authorID = db_row[5]
+            self.type = None
+            self.description = db_row[2]
+            self.created = db_row[3]
+            self.updated = db_row[4]
+            self.subjects = subjects
+        else :
+            self.courseID = None
+            self.courseName = None
+            self.authorID = None
+            self.type = None
+            self.description = None
+            self.created = None
+            self.updated = None
+            self.subjects = None
+
+    def edit(self,info):
+        if 'courseName' in info:
+            self.courseName = info['courseName']
+        if 'description' in info:
+            self.description = info['description']
+        self.updated = time.strftime("%Y-%m-%d %H:%M:%S")
+        temp = self.subjects
+        if 'subjects' in info:
+            if len(info['subjects']) == 0:
+                self.subjects = None
+            else:
+                self.subjects = info['subjects']
+        option = True
+        if temp == self.subjects:
+            option = False
+        return self.save(option)
+
+    def save(self,option):
```

Comment:
if possible rewrite it so that it doesn't contains too much indent and complex logic